content/en/device-support/_index.md

```diff
@@ -219,6 +219,29 @@ Passkeys created in **macOS** can be used on:
                                 alt="calendar icon"></i><br>Planned
                         </td>
                     </tr>
+                    <tr class="align-middle">
+                        <td class="fw-bold">Support in Native Apps</td>
+                        <td class="text-center"><i class="bi bi-wrench-adjustable-circle-fill" title="Beta"
+                                alt="wrench in circle icon"></i><br>Alpha<sup>4</sup>
+                        </td>
+                        <td class="text-center"><i class="bi bi-x-circle-fill text-danger"></i><br><span
+                                class="fs-6 text-muted">Not
+                                Supported</span></td>
+                        <td class="text-center">
+                          <i class="bi bi-check-circle-fill text-success"></i><br>
+                          <span>iOS 16+<sup>4</sup></span>
+                        </td>
+                        <td class="text-center">
+                          <i class="bi bi-check-circle-fill text-success"></i><br>
+                          <span>macOS 13+<sup>4</sup></span>
+                        </td>
+                        <td class="text-center"><i class="bi bi-x-circle-fill text-danger"></i><br><span
+                                class="fs-6 text-muted">Not
+                                Supported</span></td>
+                        <td class="text-center"><i class="bi bi-x-circle-fill text-danger"></i><br><span
```

Comment:
Passkeys are supported in native apps on Windows

Reply:
Awesome, didn't know that. 👌 I suppose it's device-bound passkeys only and https://learn.microsoft.com/en-us/windows/win32/webauthn/-webauthn-portal would be the appropriate reference I guess.